lwgenerate/general/service_account.go

```diff
@@ -0,0 +1,115 @@
+package general
```

Comment:
What's the reasoning for this package name?

Reply:
Naming be hard, but done away with it